handler/handler.go

```diff
@@ -88,6 +88,57 @@ func (h Handler) AddTask() http.HandlerFunc {
 	}
 }
 
+// GetSTDOUT fetches the stdout logs from the agent that ran the task
+func (h Handler) GetSTDOUT() http.HandlerFunc {
+	return h.getFile("stdout")
+}
+
+// GetSTDERR fetches the stderr logs from the agent that ran the task
+func (h Handler) GetSTDERR() http.HandlerFunc {
+	return h.getFile("stderr")
+}
+
+// getFile handles the actual fetching of file from the agent.
+func (h Handler) getFile(file string) http.HandlerFunc {
```

Comment:
this is not added as a handler to the http mux so I don't think it belongs as a member. Also it would make sense to split the helper for fetching from the sandbox from dealing with http junk towards the client. e.g returning the result of http.Get() directly.